### PR TITLE
Update github actions files

### PR DESCRIPTION
**Pull request by daviddavis**
[noissue]

### CODE REVIEW

**Review by dralley — APPROVED** [review]

**Review by fao89 — APPROVED** [review]

**Comment by fao89** [comment]
should I start the release after merging this PR?

**Comment by daviddavis** [comment]
@fao89 yes although you're probably going to hit a lot of breakages I imagine.

File: .github/workflows/publish_docs.yml [hunk 1]
```diff
@@ -1,47 +1,67 @@
...
+# For more info visit https://github.com/pulp/plugin_template
+---
 name: Publish docs
 
 on:
+
   push:
     branches:
       - master
     tags:
```
Comment (dralley) [inline .github/workflows/publish_docs.yml:15]: The docs are published during the release workflow so this doesn't really seem necessary? 

And publishing them on every push seems like overkill when we already are publishing them nightly

Comment (daviddavis) [reply .github/workflows/publish_docs.yml:15]: > The docs are published during the release workflow so this doesn't really seem necessary?

Agreed, I will remove that.

> And publishing them on every push seems like overkill when we already are publishing them nightly. Or at least we don't need to publish them nightly if we already do so every time the branch is updated.

So this is the current behavior in Travis. I'm not sure why we do this.

**Pull request merged**